code/src/pages/MovieDetails.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from "react"
+import { useParams } from "react-router-dom"
+
+import "../index.css"
+
+import BackButton from "../components/BackButton"
+
+export const MovieDetails = () => {
+  const { movieId } = useParams()
+  const [movieDetails, setMovieDetails] = useState([])
+
+  useEffect(() => {
+    fetch(
+      `https://api.themoviedb.org/3/movie/${movieId}?api_key=50eb4adf4920d3714372a2ce134f64ac&language=en-US`
+    )
+      .then((res) => res.json())
+      .then((json) => 
+        setMovieDetails([json])
+      )
+  }, [movieId])
+
+  return (
+    <section className="movie-details-page">
+      <BackButton />
+      {/* conditional rendering */}
+      {movieDetails.length !== 0 &&
+        movieDetails.map((details) => (
```

Comment:
It's interesting you chose to do a .map, we didn't think was necessary since the api takes the movieID as a param already.  We think you did a good job creating clean code and showing what exactly you want the map to accomplish. 😄